packages/design-system-docs/src/scripts/components/PageHeader.jsx

```diff
@@ -36,26 +63,28 @@ class PageHeader extends React.PureComponent {
   render() {
     return (
       <header className="ds-u-padding--3 ds-u-sm-padding--6 ds-u-display--block ds-u-fill--gray-lightest">
-        <h1
-          className="ds-display"
-          // eslint-disable-next-line react/no-danger
-          dangerouslySetInnerHTML={{ __html: this.props.header }}
-          id={this.props.reference}
-        />
-        <div className="ds-u-clearfix" />
-        <div className="ds-u-font-size--small">
+        <div className="ds-u-display--flex ds-u-align-items--center">
+          <h1
+            className="ds-display ds-u-display--inline-block"
+            // eslint-disable-next-line react/no-danger
+            dangerouslySetInnerHTML={{ __html: this.props.header }}
+            id={this.props.reference}
+          />
           {this.statusBadge()}
-
+        </div>
+        <div className="ds-u-font-size--small">
           {this.guidanceLink()}
-
+          {this.cmsdsLink()}
```

Comment:
Oh I completely forgot about the case where its possible to have a CMSDS link but not a USWDS, good catch. If thats the case, instead of having 3 functions for what's essentially 1 if else statement, can we organize this to be `{this.designSystemLinks()}`, where its defined as:
```
designSystemLinks () {
   const cmsdsLink = this.props.cmsds && (
       <a href={this.props.cmsds}>CMS Design System</a>
   );
   const uswdsLink = this.props.uswds && (
       <a href={this.props.uswds}>US Web Design System</a>
   );

   if (cmsds && uswdsLink) {
      return (<span> View related guidance in the {cmsdsLink} and {uswdsLink}</span>)
   } else if (cmsds) {
      return (<span> View related guidance in the {cmsdsLink}</span>)
   } else if (uswds) {
      return (<span> View related guidance in the {uswdsLink}</span>)
   } else {
      return
   }
}
```